cc/util/test_util.cc

```diff
@@ -68,6 +68,27 @@ using google::crypto::tink::Ed25519PrivateKey;
 using google::crypto::tink::Keyset;
 using google::crypto::tink::OutputPrefixType;
 
+#ifdef _WIN32
```

Comment:
minor/optional: We usually keep nested defines flat internally.

Reply:
Done.